frontend/src/utils/id.ts

```
@@ -0,0 +1,18 @@
+export const generateNextId = (items: string[], base: string) => {
+  let maxNumber = 0;
+
+  items.forEach((item) => {
+    if (item.startsWith(base)) {
+      const suffix = item.slice(base.length);
+      const match = suffix.match(/^\d+$/);
+      if (match) {
+        const num = parseInt(match[0], 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)